visual-js/replace_pkg_version.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+# Check if jq is installed
+if ! command -v jq &> /dev/null
+then
+    echo "jq could not be found. Please install jq to run this script."
+    exit
```

Comment:
```suggestion
    exit 1
```

Otherwise failures may not be catched.

Reply:
good point. done